firmware-utils: dlink-sge-image: add util to CMakeLists.txt

Signed-off-by: Daniel Linjama <daniel@dev.linjama.com>
---
 tools/firmware-utils/CMakeLists.txt        |   1 +
 1 files changed, 1 insertions(+)

diff --git a/tools/firmware-utils/CMakeLists.txt b/tools/firmware-utils/CMakeLists.txt
--- a/tools/firmware-utils/CMakeLists.txt
+++ b/tools/firmware-utils/CMakeLists.txt
@@ -38,6 +38,7 @@ FW_UTIL(buffalo-tftp src/buffalo-lib.c "" "")
 FW_UTIL(cros-vbutil "" "" "${OPENSSL_CRYPTO_LIBRARIES}")
 FW_UTIL(dgfirmware "" "" "")
 FW_UTIL(dgn3500sum "" "" "")
+FW_UTIL(dlink-sge-image "" "" "${OPENSSL_CRYPTO_LIBRARIES}")
 FW_UTIL(dns313-header "" "" "")
 FW_UTIL(edimax_fw_header "" "" "")
 FW_UTIL(encode_crc "" "" "")
